tutorials/ui/bbcode_in_richtextlabel.rst

```diff
@@ -1147,3 +1147,26 @@ This will add a few new BBCode commands, which can be used like so:
 
     [center][ghost]This is a custom [matrix]effect[/matrix][/ghost] made in
     [pulse freq=5.0 height=2.0][pulse color=#00FFAA freq=2.0]GDScript[/pulse][/pulse].[/center]
+
+Printing rich text to the console
+---------------------------------
+
+Using :ref:`print_rich<class_@GlobalScope_method_print_rich>`, you can print
+rich text to the editor Output panel and standard output (visible when the user
+runs the project from a terminal). This works by converting the BBCode to
+`ANSI escape codes <https://en.wikipedia.org/wiki/ANSI_escape_code>`__ that the
+terminal understands.
+
+In the editor output, all BBCode tags are recognized as usual. In the terminal
+output, only a subset of BBCode tags will work, as documented in the linked
+``print_rich()`` method description above. In the terminal, the colors will look
+different depending on the user's theme, while colors in the editor will use the
+same colors as they would in the project.
+
+.. note::
+
+    To print error and warning messages that can be filtered by type in the
+    editor Output panel, use
+    :ref:`@GlobalScope.push_error<class_@GlobalScope_method_push_error>` and
+    :ref:`@GlobalScope.push_warning<class_@GlobalScope_method_push_warning>`
+    instead of ``print_rich()``.
```

Comment:
I don't think this is quite technically correct in all cases, either - `push_error` and `push_warning` do print to the Output panel when the debugger is *not* running, such as in the editor. But when the debugger is running, they only appear in the debugger. You mentioned this in https://github.com/godotengine/godot-proposals/issues/10568#issuecomment-2322758852

I think this paragraph needs slight wording tweaks to be technically correct, and/or we should change the behavior of push_warning and push_error as mentioned in the linked proposal.

---

As a side note, the description in the class ref of [print_rich()](https://docs.godotengine.org/en/latest/classes/class_@globalscope.html#class-globalscope-method-print-rich) doesn't say that `push_warning` and `push_error` are always present in the Output panel, but it does slightly imply it:
> Note: Consider using [push_error](https://docs.godotengine.org/en/latest/classes/class_@globalscope.html#class-globalscope-method-push-error) and [push_warning](https://docs.godotengine.org/en/latest/classes/class_@globalscope.html#class-globalscope-method-push-warning) to print error and warning messages instead of [print](https://docs.godotengine.org/en/latest/classes/class_@globalscope.html#class-globalscope-method-print) or [print_rich](https://docs.godotengine.org/en/latest/classes/class_@globalscope.html#class-globalscope-method-print-rich). This distinguishes them from print messages used for debugging purposes, while also displaying a stack trace when an error or warning is printed.

I think the current class ref description is *fine* because it's not *directly* wrong.

Reply:
I've added a mention of this behavior in the newly added page.